src/common/transformations/tests/common_optimizations/move_eltwise_up_data_movement_test.cpp

```diff
@@ -349,3 +349,72 @@ TEST_F(MoveEltwiseUpThroughDataMovTest, TransposeFakeQuantizePerChannel) {
         manager.register_pass<ov::pass::MoveEltwiseUpThroughDataMov>();
     }
 }
+
+TEST_F(MoveEltwiseUpThroughDataMovTest, PerChannelEltwiseUnsqueeze) {
```

Comment:
please add a test for BinaryElementwiseArithmetic(eltw_const_in, eltw_data_flow_in) case

Reply:
Added